### ISSUE TITLE
Asset transactions table update

### ISSUE THREAD

**Issue by sonam-jo** [centrifuge/apps#2526]
Data --> Asset transactions table 

![Image](https://github.com/user-attachments/assets/a03b1614-f8fe-4cd1-892c-d8076faf03d2)

Can we change the format of this table to be the same as the `Transaction history` table on the `Overview` page with:
- Transaction date
- Transaction
- Amount
- Epoch (we keep this from the current asset transactions table)
- View transaction

![Image](https://github.com/user-attachments/assets/31b49c0e-4651-49d0-a3e6-410197c8c1d3)

**Comment by 0x4Graham** [comment]
@kattylucy Is it possible to know what asset was purchased? Like we currently have in the transaction history table? I feel we should extend this to include the asset.  @sonam-jo what do you think? 

<img width="932" alt="Image" src="https://github.com/user-attachments/assets/4d2b2fd8-c639-4162-aed0-2f71df130fbd" />

**Comment by sonam-jo** [comment]
Yes, similar to how we have the table in Transaction history on the Overview page, we need to have `Purchase of X asset`, `Sale of x asset`, `Financing of Y asset` under the `Transaction` column

**Comment by kattylucy** [comment]
<img width="1061" alt="Image" src="https://github.com/user-attachments/assets/a3d5698d-035a-47da-8eea-49e483b9787b" />

**Comment by kattylucy** [comment]
The filters not longer match the table anymore - which filters should we add or do I just remove it  and keep from/to @sonam-jo
